### PR TITLE
Software table

### PR DESCRIPTION
**Pull request by jeromekelleher**
Another pass at the software table. Looks like:

demes-table

We can tidy it up a bit, but I think the format is good. It's a nice concise way of getting across the current level of support.

### CODE REVIEW

**Comment by jeromekelleher** [comment]
If we like this, I'll create issues to get people to update the various rows appropriately.

File: paper.tex [hunk 1]
```diff
@@ -137,46 +138,71 @@ \subsection*{Unambiguous low-level representation}
...
-    & Demographic inference \\
-\dadi & \citep{gutenkunst2009inferring}  & Demographic inference \\
-\gadma & \citep{noskova2020gadma}  & Demographic inference \\
+\begin{tabular}{lp{12cm}}
+\toprule
+\multicolumn{2}{l}{Software infrastructure}\\
+\midrule
+\texttt{demes-python} &
+    A Python library for loading, saving, and working with
+    Demes models. Includes conversion utilities for existing formats
```
Comment (grahamgower) [inline paper.tex]:
```suggestion
    Demes models. Includes conversion utilities for existing
```

File: paper.tex [hunk 2]
```diff
@@ -137,46 +138,71 @@ \subsection*{Unambiguous low-level representation}
...
+    Compute expected diversity statistics (SFS, LD) and perform
+    demographic inference using summary statistics
+    \citep{jouganous2017inferring,ragsdale2019models}.\\
+
+\msprime &
+    Coalescent simulation of Demes models, including command line
+    interface~\citep{kelleher2016efficient,kelleher2020coalescent}.\\
+
+\stdpopsim &
+    A collection of community empirical data for species
```
Comment (grahamgower) [inline paper.tex]:
```suggestion
    A community-maintained collection of empirical data for species
```

**Review by grahamgower — APPROVED** [review]
LGTM. I made some small edits to the wording.

**Pull request merged**